pkg/controller/servicebindingrequest/binder_test.go

```diff
@@ -55,60 +54,75 @@ func TestBinderNew(t *testing.T) {
 
 	t.Run("search target object by resource name", func(t *testing.T) {
 		list, err := binderForSBRWithResourceRef.search()
-		assert.Nil(t, err)
-		assert.Equal(t, 1, len(list.Items))
+		require.NoError(t, err)
+		require.Equal(t, 1, len(list.Items))
 	})
 
 	t.Run("search", func(t *testing.T) {
 		list, err := binder.search()
-		assert.Nil(t, err)
-		assert.Equal(t, 1, len(list.Items))
+		require.NoError(t, err)
+		require.Equal(t, 1, len(list.Items))
 	})
 
 	t.Run("appendEnvFrom", func(t *testing.T) {
 		secretName := "secret"
 		d := mocks.DeploymentMock("binder", "binder", map[string]string{})
 		list := binder.appendEnvFrom(d.Spec.Template.Spec.Containers[0].EnvFrom, secretName)
 
-		assert.Equal(t, 1, len(list))
-		assert.Equal(t, secretName, list[0].SecretRef.Name)
+		require.Equal(t, 1, len(list))
+		require.Equal(t, secretName, list[0].SecretRef.Name)
 	})
 
 	t.Run("appendEnv", func(t *testing.T) {
 		d := mocks.DeploymentMock("binder", "binder", map[string]string{})
 		list := binder.appendEnvVar(d.Spec.Template.Spec.Containers[0].Env, "name", "value")
-		assert.Equal(t, 1, len(list))
-		assert.Equal(t, "name", list[0].Name)
-		assert.Equal(t, "value", list[0].Value)
+		require.Equal(t, 1, len(list))
+		require.Equal(t, "name", list[0].Name)
+		require.Equal(t, "value", list[0].Value)
```

Comment:
(See the previous comment) The length could be a `require` and the other two could be `assert`.

Reply:
This PR is already big enough, we should add an issue to evaluate our usage of `require` and `assert` in tests and deal with those as another work unit.

What do you think?

Reply:
I am fine with that.